searches/search_quadruplet.py

```diff
@@ -0,0 +1,72 @@
+def search_quadruplets(sequence: list[int], target: int) -> list[int]:
```

Comment:
```suggestion
def search_quadruplets(sequence: list[int], target: int) -> list[list[int]]:
```
Return type is incorrect